<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenAccount</class>
 <widget class="QDialog" name="FenAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agenda-Online - Gestion des comptes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="nomDeCompteLabel">
       <property name="text">
        <string>Nom de compte: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="accountLine"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="motDePasseLabel">
       <property name="text">
        <string>Mot de passe:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordLine">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="classeLabel">
       <property name="text">
        <string>Classe: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="classeLine"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="iDLabel">
       <property name="text">
        <string>ID: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="IdLine">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QCheckBox" name="useIDForDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Utiliser l'ID pour supprimer</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QCheckBox" name="useIDForInterrogation">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Utiliser l'ID pour interroger</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="execute">
     <property name="text">
      <string>Ex√©cuter l'action choisie.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="add">
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="del">
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="set">
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="interrogate">
     <property name="text">
      <string>Interroger</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
